src/main/java/org/broadinstitute/consent/http/service/NihService.java

```diff
@@ -51,8 +49,11 @@ public  User syncAccount(DuosUser duosUser) throws Exception {
       String body = response.parseAsString();
       NIHUserAccount nihAccount = parseNihUserAccount(body);
       serviceDAO.updateUserNihStatus(user, nihAccount);
-    } catch (NotFoundException e) {
+    } catch (NotFoundException _) {
       serviceDAO.deleteNihAccountById(user.getUserId());
+    } catch (NotAuthorizedException _) {
+      // ECM will return a 401 if the user has not accepted ToS yet.
+      logWarn("ECM Response: not authorized user: %s".formatted(duosUser.getEmail()));
```

Comment:
I'm wondering if the log should include why this one is expected.
```suggestion
      logWarn("ECM Response: not authorized user: %s. User needs to accept the ToS.".formatted(duosUser.getEmail()));
```